wremnants/histselections.py

```diff
@@ -24,12 +24,25 @@ def fakeHistIsoRegionIntGen(h, scale=1.):
     print("Slicing")
     return h[{"iso" : 0, "qTgen" : s[::hist.sum]}]
 
-def signalHistWmass(h, charge=None):
-    sel = {"passIso" : 1, "passMT" : 1}
+def signalHistWmass(h, charge=None, passIso=1, passMT=1):
```

Comment:
Why 0,1 instead of true/false?

Reply:
No reason, maybe I just took it from other functions which use 0 and 1, I can change it